pipeline/metering/shadow_evidence.py

```
@@ -0,0 +1,201 @@
+        errs = []
+        base_recs = []
+        if os.path.isfile(args.base):
+            errs.extend(verify_file(args.base))
+            base_recs = [json.loads(l) for l in read_lines(args.base)]
+        errs.extend(verify_file(args.local))
+        if errs:
+            for e in errs[:10]:
+                print(f"CHAIN {e}", file=sys.stderr)
+            die(3, "基链或本地影子片验链失败——拒绝合并（防覆盖掩盖篡改）")
+        merged, prev = list(base_recs), None
+        for rec in [json.loads(l) for l in read_lines(args.local)]:
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)